src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPService.kt

```diff
@@ -83,6 +84,7 @@ private fun CSIPReport.toCSIPResponse(): CSIPResponse =
     decision = toDecisionResponse(),
     plans = plans.map { it.toPlanResponse() },
     reviews = reviews.map { it.toReviewResponse() },
+    documents = offenderBooking.documents.map { it.toDocumentResponse() },
```

Comment:
sure this would return all documents not just CSIP documents?
Don't you need to filter or call a new repository method?